Documentation/minio-object-store.md

```diff
@@ -65,15 +65,15 @@ Minio comes with an embedded web based object browser. In the example, the objec
 which port has been assigned to the service via:
 
 ```console
-kubectl -n rook-minio get service minio-service -o jsonpath='{.spec.ports[0].nodePort}'
+kubectl -n rook-minio get service my-store-minio-service -o jsonpath='{.spec.ports[0].nodePort}'
```

Comment:
Instead of a prefix, I would suggest a suffix for the instance name of the service or other resources. When viewing resources with kubectl I think it lists them in a more predictable order and it will be easier to see all of the minio resources grouped together even if there are other resources in the namespace.

Reply:
I renamed it to `my-store-minio` (the `-service` is redundant), as the statefulset Pods are named `my-store-ID`.